pkg/crc/validation/validation.go

```diff
@@ -85,6 +75,21 @@ func ValidateBundle(bundlePath string, preset crcpreset.Preset) error {
 	return nil
 }
 
+func bundleMismatchWarning(userProvidedBundle string, preset crcpreset.Preset) {
+	userProvidedBundle = bundle.GetBundleNameWithExtension(userProvidedBundle)
+	if userProvidedBundle != constants.GetDefaultBundle(preset) {
+		warnMsg := fmt.Sprintf("Using %s bundle, but %s is expected for this release", userProvidedBundle, constants.GetDefaultBundle(preset))
```

Comment:
@cfergeau if I don't use `warnMsg` then I need to use `if <-> else` block because for custom bundle we don't want to put the warning that it is not supported but just show it is custom bundle.
```golang
	if userProvidedBundle != constants.GetDefaultBundle(preset) {
		constants.GetDefaultBundle(preset))
		// Should append underscore (_) here, as we don't want crc_libvirt_4.7.15.crcbundle
		// to be detected as a custom bundle for crc_libvirt_4.7.1.crcbundle
		usingCustomBundle := strings.HasPrefix(bundle.GetBundleNameWithoutExtension(userProvidedBundle),
			fmt.Sprintf("%s_", bundle.GetBundleNameWithoutExtension(constants.GetDefaultBundle(preset))))
		if usingCustomBundle {
			logging.Warnf("Using custom bundle %s", userProvidedBundle)
		} else {
                        logging.Warnf("Using %s bundle, but %s is expected for this release", userProvidedBundle, 
                }
	}
```

Reply:
Yes, this `if/else` block looks easy enough to read?